### PR TITLE
removed redundancy in userinfo stored in mongoDb

### PR DESCRIPTION
**Pull request by grvsood**
### Why
Currently, we make the post request in the SignIn method to MongoDB which leads to redundancy in user data in the database

### Changes Done
UserInfo to be stored only once i.e. during signUp

### Testing
npm run serve (in root directory)
localhost:3000/api/users/

### CODE REVIEW

**Comment by grvsood** [comment]
#44

File: client/src/store/modules/user.js [hunk 1]
```diff
@@ -48,7 +48,6 @@ export default {
           userInfo = response.data.filter(x => x.id === user.user.uid)
```
Comment (DavidGolodetsky) [inline client/src/store/modules/user.js:48]: It's related to previous commit, just a suggestion. Would be great to extend Node.js route and find user by Id on that level. I that case we would fetch particular user on backend level and wouldn't need to query all users and filter them. Example could be found here GeekScore-2/routes/api/games.js where we fetch particular game by id. Again could be another PR if you want to cover this issue

**Review by DavidGolodetsky — APPROVED** [review]
